src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -169,18 +169,22 @@ import { appRouter } from '../appRouter';
 
         elem.querySelector('.previousTrackButton').addEventListener('click', function (e) {
             if (currentPlayer) {
-                if (lastPlayerState.NowPlayingItem.MediaType === 'Audio' && (currentPlayer._currentTime >= 5 || !playbackManager.previousTrack(currentPlayer))) {
-                    // Cancel this event if doubleclick is fired
-                    if (e.detail > 1 && playbackManager.previousTrack(currentPlayer)) {
+                if (lastPlayerState.NowPlayingItem.MediaType === 'Audio') {
+                    // Cancel this event if doubleclick is fired. The actual previousTrack will be processed by the 'dblclick' event
+                    if (e.detail > 1 ) {
+                        return;
+                    }
+                    // Return to start of track, unless we are already (almost) at the beginning. In the latter case, continue and move
+                    // to the previous track, unless we are at the first track so no previous track exists.
+                    if (currentPlayer._currentTime >= 2 || playbackManager.getCurrentPlaylistIndex(currentPlayer) <= 1) {
```

Comment:
Why the change from 5 to 2 here? That seems like it could be a little low.

Reply:
Ok, thanks for that! My interpretation was that the 5 seconds from the start of a tune are meant to 'simulate' a long 'doubleclick', but it felt like a little too long to me as doubleclick.
But maybe I am just impatient, or maybe there is a technical reason for a 'long' doubleclick, so no problem of course to use 5 seconds after all.
I'll submit an update.